packages/x-tree-view-pro/package.json

```diff
@@ -51,7 +51,9 @@
     "@types/react-transition-group": "^4.4.11",
     "clsx": "^2.1.1",
     "prop-types": "^15.8.1",
-    "react-transition-group": "^4.4.5"
+    "react-transition-group": "^4.4.5",
+    "reselect": "^5.1.1",
+    "use-sync-external-store": "^1.0.0"
```

Comment:
Required to keep supporting React 17
Used by Base UI [here](https://github.com/mui/base-ui/blob/b37dfc96e1a7d2515df19e3cafa5963039951257/packages/mui-base/src/useMediaQuery/index.ts#L2C10-L2C30)